src/report.rs

```diff
@@ -134,7 +135,8 @@ pub fn get_report(args: ReportArgs, hv: bool) -> Result<()> {
         .write(true)
         .open(&args.att_report_path)?;
 
-    report.write_bytes(&mut file)?;
+    let bytes = report.to_bytes()?.to_vec();
+    file.write_all(&bytes)?;
```

Comment:
We can use the new function encode to write directly to the writers
```suggestion
   report.encode(&mut file, ())?;
```